crates/store/src/state.rs

```diff
@@ -784,7 +784,7 @@ impl State {
     }
 
     /// Returns data needed by the block producer to verify transactions validity.
-    #[instrument(target = COMPONENT, skip_all, ret)]
+    #[instrument(level = "debug", target = COMPONENT, skip_all, ret)]
```

Comment:
I think this can remain `info`. Its used by the block-producer when a transaction is submitted, to get its related state.

Counter-intuitively, this is not actually used by the client etc, and is not used by users themselves. This can be verified by checking the RPCs proto definition which _does not_ include this method.

Partially why I want to separate the gRPC definitions for the store out :)

```suggestion
    #[instrument(target = COMPONENT, skip_all, ret)]
```